### DISCUSSION TITLE
Highlighted grid line color

### DISCUSSION THREAD

**Discussion by biggysmith** [epezent/implot#596] [Q&A]
Hi all,

Im trying to disable the highlighted gridline that appears every 10th, 100th etc line:

![image](https://github.com/user-attachments/assets/1e215570-2614-49d5-85fa-542f60ca7767)

I don't know what its called to change.  Any Ideas?

**Comment by hinxx** [comment]
Does changing the `implot.cpp` line 1808 to be the same as 1810 solve it for you?

**Reply by biggysmith** [reply]
I would expect so, but I'm using implot through python so its kinda tricky to change that manually.

Would be great if it there was a option to control it.

**Reply by hinxx** [reply]
Try to change the style of MajorGridSize to match MinorGridSize. They are part of the ImPlotStyle struct; the implot demo window allows you to tinker with those settings. 

You'll probably have to look into the c++ sources for more guidance.

**Reply by biggysmith** [reply]
This sorta works as a rough workaround:

```
implot.push_style_var(implot.StyleVar_.minor_alpha, 1.0)      
implot.push_style_color(implot.Col_.axis_grid, [1.0, 1.0, 1.0, 0.0625])

...

implot.pop_style_color()     
implot.pop_style_var()
```

![image](https://github.com/user-attachments/assets/258cacb5-4fb8-426a-ad4e-b71481318dbb)

although the ticks are faded now.
